kalite/distributed/backup.py

```diff
@@ -0,0 +1,36 @@
+import dbbackup
+import datetime
+import os
+import sys
+import time
+from os.path import expanduser
```

Comment:
you don't have to import these, you can actually just `os.path.join` etc after importing os.